packages/jest-jasmine2/src/index.js

```diff
@@ -44,6 +45,18 @@ async function jasmine2(
   Object.assign(environment.global, jasmineInterface);
   env.addReporter(jasmineInterface.jsApiReporter);
 
+  if (config.testLocationInResults === true) {
+    const originalIt = environment.global.it;
+    environment.global.it = (...args) => {
+      const stack = callsites()[1];
```

Comment:
not sure about hardcoding the index. thoughts? Can traverse all and returning the first one which is not inside `node_modules` instead, perhaps?